frontend/apps/app/components/Chat/AgentMessage/AgentMessage.tsx

```diff
@@ -69,7 +68,6 @@ export const AgentMessage: FC<AgentMessageProps> = ({
         <span className={styles.agentName}>
           {agentName || (isAsk ? 'Ask Agent' : 'Build Agent')}
         </span>
-        {time && <span className={styles.messageTime}>{time}</span>}
```

Comment:
I noticed that message.created_at is already being passed in the next pull request, so it looks like this removal isn’t necessary.

frontend/apps/app/components/Chat/services/messageServiceClient.ts:122
```
timestamp: new Date(message.created_at),
```

Reply:
Thank you!
I believe it's something I temporarily removed to deal with a hydration error. I'll revisit it later!